packages/react-native/ReactAndroid/src/test/java/com/facebook/react/uimanager/SimpleViewPropertyTest.kt

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.react.uimanager
+
+import android.graphics.drawable.ColorDrawable
+import android.view.View
+import com.facebook.react.bridge.CatalystInstance
+import com.facebook.react.bridge.JavaOnlyMap
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReactTestHelper.createMockCatalystInstance
+import com.facebook.react.bridge.ReadableMap
+import com.facebook.react.touch.JSResponderHandler
+import com.facebook.react.uimanager.annotations.ReactProp
+import org.assertj.core.api.Assertions
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.powermock.core.classloader.annotations.PowerMockIgnore
+import org.powermock.modules.junit4.rule.PowerMockRule
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+
+/** Verify [View] view property being applied properly by [SimpleViewManager] */
+@RunWith(RobolectricTestRunner::class)
+@PowerMockIgnore("org.mockito.*", "org.robolectric.*", "androidx.*", "android.*")
+class SimpleViewPropertyTest {
+
+  @get:Rule var rule = PowerMockRule()
+
+  private class ConcreteViewManager : SimpleViewManager<View?>() {
+    @ReactProp(name = "foo") fun setFoo(view: View, foo: Boolean) {}
+
+    @ReactProp(name = "bar") fun setBar(view: View, bar: ReadableMap?) {}
+
+    override fun createViewInstance(reactContext: ThemedReactContext): View {
+      return View(reactContext)
+    }
+
+    override fun getName(): String {
+      return "View"
+    }
+  }
+
+  private lateinit var mContext: ReactApplicationContext
```

Comment:
NIT: let's rename:

mContext -> context
mCatalystInstanceMock -> catalystInstanceMock
and so on...